Export Detailed Child Usages release notes (#3109)
SumoLogic/sumologic-documentation: blog-service/2023-09-25-manage-account.md

---
title: Sumo Orgs - Export Detailed Child Usages (Manage Account)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - manage
  - child-usage
  - account
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

import useBaseUrl from '@docusaurus/useBaseUrl';

> We're excited to introduce a new addition to Sumo Logic Orgs parent account users in our accounts page, you can now export/download detailed child org usage data directly from the account overview page.

Reviewer: ```suggestion
We're excited to introduce a new addition to Sumo Logic account management. Org parent account users can now download detailed child org usage data directly from the account overview page.
```
